ADOPTERS.md

```diff
@@ -0,0 +1,24 @@
+# Adopters
+
+This is a page listing the adopters of the OpenZiti project along with a brief description of what the project/product offers.
+
+## Adding Your Project to the List
+
+If you're interested in getting your project added to the list either:
```

Comment:
"either" seems wrong here

Reply:
i forgot to add "put up a pull request editing the table below". thx i will fix that.

Reply:
added to bullets